### PR TITLE
Add Screenshot-secion to IssueTemplate.md

### PR DESCRIPTION
**Pull request by bkcsoft**
For `UI`-bugs we usually require a Screenshot to debug it :wink:

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -21,3 +21,8 @@
 ## Description
 
 ...
+
+
+## Screenshots
+
+**If regarding the Web Interface, please include a screenshot**
```
Comment (andreynering) [inline .github/issue_template.md:28]: Does this need to be bold?

**Comment by andreynering** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Comment by lunny** [comment]
Why `continuous-integration/drone/push` is always failed?

**Pull request merged**